packages/instant-meilisearch/package.json

```diff
@@ -104,6 +82,7 @@
     "shx": "^0.3.3",
     "ts-jest": "^27.0.7",
     "tslib": "^2.4.1",
-    "typescript": "^4.9.3"
+    "typescript": "^4.9.3",
+    "eslint-config-meilisearch": "*"
```

Comment:
Why is it not `@meilisearch/eslint-config`? 😄

Reply:
It's because of the eslint convention naming. If tomorrow I want to publish this configuration to npm it needs to follow that naming convention